test/e2e/feature/feature.go

```diff
@@ -60,12 +60,6 @@ var (
 	// Owner: sig-autoscaling
 	ClusterSizeAutoscalingScaleUp = framework.WithFeature(framework.ValidFeatures.Add("ClusterSizeAutoscalingScaleUp"))
 
-	// TODO: document the feature (owning SIG, when to use this feature for a test)
-	ClusterTrustBundle = framework.WithFeature(framework.ValidFeatures.Add("ClusterTrustBundle"))
```

Comment:
Where were these tests running before?

Reply:
I think https://github.com/kubernetes/test-infra/pull/34451 lists all the jobs. I don't think there are any feature-specific setup steps required - these featuregates only need to be applied to kube-apiserver, KCM and kubelet.